101-synapse-poc/azuredeploy.json

```diff
@@ -290,7 +290,7 @@
             "properties": {
                 "mode": "Incremental",
                 "templatelink": {
-                    "uri": "https://raw.githubusercontent.com/JamJarchitect/azure-quickstart-templates/master/101-synapse-poc/nestedtemplates/pausetemplate.json"
+                    "uri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-synapse-poc/nestedtemplates/pausetemplate.json"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle nested templates